bookmark-it/manifest.json

```diff
@@ -0,0 +1,36 @@
+{
+  "manifest_version": 2,
+  "name": "Bookmark it!",
+  "version": "1.0",
+  "description": "A simple bookmark button",
+  "homepage_url": "https://github.com/mdn/webextensions-examples/tree/master/bookmark-it",
+  "icons": {
+    "48": "icons/bookmark-it.png",
+    "96": "icons/bookmark-it@2x.png"
+  },
+
+  "manifest_version": 2,
+  "name": "bookmark-it",
+  "version": "1.0",
+
+  "applications": {
+    "gecko": {
+      "id": "bookmark-it-extension@mozilla.org"
```

Comment:
Good question. Maybe add a notice that it will work only from Firefox 47 onwards, then it will not get outdated?